script/upload

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -x
+
+export PATH=/snap/bin:$PATH
+
+charm whoami
+RET=$?
+if ((RET > 0)); then
+    echo "Not logged into charmstore"
+    exit 1
+fi
+
+rm -rf temp
+mkdir temp
+cd temp
+unzip ../"$CHARM".charm
+URL=$(charm push . cs:~"$NAMESPACE"/"$CHARM" | yq r - url)
+charm attach cs:~"$NAMESPACE"/"$CHARM" --channel unpublished multus-image=nfvpe/multus:v3.4
```

Comment:
@Cynerva I'll need some guidance on what to do here to make sure the uploads work, it needs some resource settings set

Reply:
During the build script, you'll need to:
1. `docker pull nfvpe/multus:v3.4` to make that image available for charm attach.
2. `cd net-attach-def-manager && ./build` to build the net-attach-def-manager image.

During the upload script, you'll need to:
1. `charm attach cs:~"$NAMESPACE"/"$CHARM" --channel unpublished net-attach-def-manager-image=net-attach-def-manager`.
2. update the `charm release` to include the net-attach-def-manager-image resource.